app/deck/configuration.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Configuration
+---
+
+decK can be configured using CLI flags, environment variables, and a configuration file on disk.
+
+decK reads these configuration sources in the following priority order:
+1. CLI flags override all other values.
+2. Environment variables override the configuration file.
+3. Configuration file values are treated as defaults.
+
+## CLI flags
+
+To view the available CLI flags for a command, run `deck <command> --help`. 
+You can provide these flags when running the command. For example:
+
+```bash
+deck gateway sync --konnect-control-plane-name demo --konnect-token $KONNECT_TOKEN kong.yaml
+```
+
+## Environment variables
+
+All CLI flags can also be provided as environment variables.
+
+To determine the environment variable name, replace the `.` and `-` characters in the CLI flag with `_` and prefix it with `DECK_`.
+
+To run the same `deck gateway sync` command as above using enviroment variables:
+
+```bash
+export DECK_KONNECT_CONTROL_PLANE_NAME=demo
+export DECK_KONNECT_TOKEN=$KONNECT_TOKEN
+deck gateway sync kong.yaml
+```
```

Comment:
I'd like to see an example of how these decK env var can be referenced in a decK file. Ex. `${{ env "DECK_API_KEY" }}`

Reply:
You wouldn't refer to these specific values in a decK file as they're used to configure the CLI.

Does http://localhost:8888/deck/gateway/sensitive-data/#using-deck-environment-variables meet your needs?